### PR TITLE
[DOCFIX] Update Caching Docs

### PR DESCRIPTION
**Pull request by HelloHorizon**

### CODE REVIEW

File: docs/en/core-services/Caching.md [hunk 1]
```diff
@@ -88,14 +88,14 @@ alluxio.worker.tieredstore.level0.dirs.mediumtype=MEM,SSD,SSD
 ```
 
 Note that the ordering of the medium types must match with the ordering of the paths.
-Here, MEM and SSD are two types preconfigured in Alluxio.
+Here, MEM and SSD are two types of preconfigured in Alluxio.
```
Comment (gpang) [inline docs/en/core-services/Caching.md:91]: I think this works better:

```suggestion
Here, MEM and SSD are two preconfigured types in Alluxio.
```

Comment (HelloHorizon) [reply docs/en/core-services/Caching.md]: changed.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**